provider/resource_corp_cloudwaf_instance.go

```diff
@@ -69,6 +70,15 @@ func resourceCorpCloudWAFInstance() *schema.Resource {
 							Type:        schema.TypeString,
 							Description: `Set instance location to "direct" or "advanced".`,
 							Required:    true,
+							ValidateFunc: func(val interface{}, key string) (warns []string, errs []error) {
+								if val == nil {
+									return nil, nil
+								}
+								if existsInString(val.(string), "direct", "advanced") {
+									return nil, nil
+								}
+								return nil, []error{fmt.Errorf("received instance_location '%s' is invalid. should be 'direct' or 'advanced'", val.(string))}
```

Comment:
```suggestion
								return nil, []error{fmt.Errorf(`received instance_location %q is invalid. should be "direct" or "advanced"`, val.(string))}
```

Small thing but I'd like to use double quotes here since single quotes aren't valid in TF.